common/Cargo.toml

```diff
@@ -18,7 +18,7 @@ anyhow = "1.0"
 bech32 = "0.9"
 generic-array = "0.14"
 lazy_static = "1.4"
-merkletree = "0.22"
+merkletree = "0.22.1"
```

Comment:
Instead of specifying the version manually why not run cargo update to do that for you?

Reply:
I don't wanted to change the versions of other dependencies, but I have run `cargo update` now.